catroid/src/main/res/values-th/strings.xml

```diff
@@ -1652,7 +1652,7 @@ A-8010 กราซ, ออสเตรีย \n
     <string name="formula_editor_logic_comparison">ผู้ประกอบการเปรียบเทียบ</string>
     <string name="formula_editor_object_general">คุณสมบัติทั่วไป</string>
     <string name="formula_editor_object_movement">คุณสมบัติเคลื่อนที่</string>
-    <string name="formula_editor_device_visual_detection">เซ็นเซอร์ภาพ</string>
+    <string name="formula_editor_device_face_detection">เซ็นเซอร์ภาพ</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations